Overview.src.html

```diff
@@ -4113,8 +4114,9 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   </ol>
 </ol>
 
-<p>The <dfn title=dom-Headers><code>Headers(<var>init</var>)</code></dfn> constructor,
-when invoked, must run these steps:
+<p>The
+<dfn title=dom-Headers data-export data-dfn-for=Headers data-dfn-type=constructor data-lt="Headers(init)"><code>Headers(<var>init</var>)</code></dfn>
```

Comment:
> Is that `lt` really needed?

I don't know, honestly. It's not clear to me whether the backend needs a clean `lt`, or whether it parses out data from the tag, even if the value includes HTML. *shrug* @tabatkins would know, I'm sure.

> Also, constructors and methods generally shouldn't be referenced elsewhere. Maybe in examples and such...

As above, I'd err on the side of exporting everything, but _I_ don't personally need any of this, so...